app/.env.example

```
@@ -1,4 +1,7 @@
-#url
+# API URL
 REACT_APP_BASE_HREF="http://localhost:8848"
 REACT_APP_BASE_API_URL="http://localhost:8848/api"
-REACT_APP_GOOGLE_LOGIN_URL="http://localhost:8848/api/auth/google"
\ No newline at end of file
+REACT_APP_GOOGLE_LOGIN_URL="http://localhost:8848/api/auth/google"
+
+# Google Client ID KEY 
+REACT_APP_GOOGLE_CLIENT_ID=
```

```suggestion
REACT_APP_GOOGLE_CLIENT_ID=${REACT_APP_GOOGLE_CLIENT_ID}
```
${} will make easier to use envsub to substitute the env with the given value in CI/CD tool.